AGENT_CHANGELOG.md

```diff
@@ -916,7 +944,7 @@
 * HDFS Namenode [2.2.0](https://github.com/DataDog/integrations-core/blob/master/hdfs_namenode/CHANGELOG.md)
 * Hive [1.7.1](https://github.com/DataDog/integrations-core/blob/master/hive/CHANGELOG.md)
 * HiveMQ [1.5.1](https://github.com/DataDog/integrations-core/blob/master/hivemq/CHANGELOG.md)
-* HTTP [6.1.2](https://github.com/DataDog/integrations-core/blob/master/http_check/CHANGELOG.md)
+* HTTP [6.1.2-rc.1](https://github.com/DataDog/integrations-core/blob/master/http_check/CHANGELOG.md)
```

Comment:
Wasn't sure if this should be there, since technically the `7.33.0` tag of Agent Release has HTTP check version 6.1.2-rc.1, and the previous [7.44 release changed this](https://github.com/DataDog/integrations-core/pull/14473/files#diff-79e522491d45b8d05412a8555a7eb1de83e3816d71b24090475d3dfa4b34006eL2589).